docs/ARCHITECTURE.md

```diff
@@ -9,10 +23,34 @@ can be configured to run only during certain time periods or not at all. Setting
 make sure repair is spread out over the interval time while alarms are provided to signal when a job has not run for
 longer than expected.
 
+<figure>
+  <div align="center">
+    <img src="images/ecchronos-instances.png" alt="Alt Text" style="display: block; margin-left: auto; margin-right: auto; width: 500px;">
+    <br/>
+    <figcaption >Figure 1: ecChronos and Cassandra Nodes.</figcaption>
+  </div>
+</figure>
+
 ## Concepts
 
 ### Leases
 
+In the context of Apache Cassandra, a "lease" refers to a mechanism employed during the repair process within Cassandra.
```

Comment:
I find this information a bit too generic, this document should be in the context of ecChronos not Cassandra. We already explain how ecChronos achieves coordination on the locks(leases) below. Maybe just adding a reference would be enough.